docs/sources/panels-visualizations/visualizations/heatmap/index.md

```diff
@@ -114,6 +114,7 @@ Use these settings to refine your visualization.
 
 - **Show tooltip -** Show heatmap tooltip.
 - **Show Histogram -** Show a Y-axis histogram on the tooltip. A histogram represents the distribution of the bucket values for a specific timestamp.
+- **Show color scale -** Show a color scale on the tooltip. The color scale represents the mapping between bucket value and color.
```

Comment:
This UI option will only show up if FF is enabled - maybe we should make note of that here (I think there are examples of this in the docs somewhere 🤔)